ios/Classes/Keys.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Keys.swift
+//  customer_io
+//
+//  Created by ShahrozAli on 11/11/22.
+//
```

Comment:
Can be ignored right now, but we would need to remove the headers across all files before final release.

Reply:
The Swift format and swift lint tools that we use on iOS SDK do this for us. 

We can configure git hooks like the Android SDK in this project that run the linter/formatter against this code?

Reply:
Yup will move on to `lint/hooks/ci/cd` after the basic implementation is done. Since i am the only one working, i dont get any conflicts as such either.